### DISCUSSION TITLE
how can delete record regarding label/labeler from my DID Document in partial?

### DISCUSSION THREAD

**Discussion by itaru2622** [bluesky-social/atproto#2405] [How to do it?]
When I tested with temporal self-hosting ozone, additional record for labeling was added in my DID document as below.
How can delete the record regarding label/labeler from my DID Document in partial?

i.e I'd like to finish my test and shutdown my self-hosting ozone, and clear garbage.

```
{
  "@context": [    "https://www.w3.org/ns/did/v1", ...  ],
  "id": "did:plc:...",
  "alsoKnownAs": [    "at://..."  ],
  "verificationMethod": [
    :
    { "id": "did:plc:...#atproto_label",
      "type": "Multikey",
      "controller": "did:plc:...",
      "publicKeyMultibase": "..."
    }
  ],
  "service": [
    :
    {  "id": "#atproto_labeler",
      "type": "AtprotoLabeler",
      "serviceEndpoint": "https://temporal-self-hosting-ozone-URL"
    }
  ]
}
```

**Comment by bnewbold** [comment]
There isn't a very easy or accessible way to do this right now. Technically, you can do it using the PLC identity update endpoints documented in the account migration flow:
https://github.com/bluesky-social/pds/blob/main/ACCOUNT_MIGRATION.md

**Reply by itaru2622** [reply]
@bnewbold  thank you for response. I will check and try it.

**Comment by merill** [comment]
There is an easy way now. 

Use https://www.npmjs.com/package/@skyware/labeler

and run 

```
npx @skyware/labeler clear  
```

**Reply by itaru2622** [reply]
@merill  thank you. I will try it.

**Reply by merill** [reply]
I just had to do it with my account and it worked!
